### PR TITLE
Profile in messageRequest

### PR DESCRIPTION
**Pull request by Bilb**

### CODE REVIEW

File: ts/session/messages/outgoing/controlMessage/MessageRequestResponse.ts [hunk 1]
```diff
@@ -1,18 +1,47 @@
...
+      } else {
+        this.profileKey = new Uint8Array(
+          ByteBuffer.wrap(params.lokiProfile.profileKey).toArrayBuffer()
+        );
+      }
+    }
+
+    this.displayName = params.lokiProfile && params.lokiProfile.displayName;
+
+    // no need to iclude the avatarPointer if there is no profileKey associated with it.
```
Comment (yougotwill) [inline ts/session/messages/outgoing/controlMessage/MessageRequestResponse.ts:40]: This code seems to be duplicated inside of ts/session/messages/outgoing/visibleMessage/VisibleMessage.ts. Can we make it into it's own function / check?

**Review by yougotwill — CHANGES_REQUESTED** [review]

File: ts/session/messages/outgoing/controlMessage/MessageRequestResponse.ts (hunk 1 above)
Comment (Bilb) [reply ts/session/messages/outgoing/controlMessage/MessageRequestResponse.ts]: amended

File: ts/test/session/unit/messages/MessageRequestResponse_test.ts [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+      'https://somevalidurl.com'
+    );
+    // don't ask me why deep.eq ([1,2,3, ...]) gives nothing interesting but a 8192 buffer not matching
+    expect(decoded.messageRequestResponse?.profileKey.length).to.be.eq(6);
+    expect(decoded.messageRequestResponse?.profileKey[0]).to.be.eq(1);
+    expect(decoded.messageRequestResponse?.profileKey[1]).to.be.eq(2);
+    expect(decoded.messageRequestResponse?.profileKey[2]).to.be.eq(3);
+    expect(decoded.messageRequestResponse?.profileKey[3]).to.be.eq(4);
+    expect(decoded.messageRequestResponse?.profileKey[4]).to.be.eq(5);
+    expect(decoded.messageRequestResponse?.profileKey[5]).to.be.eq(6);
```
Comment (yougotwill) [inline ts/test/session/unit/messages/MessageRequestResponse_test.ts:117]: lol

**Review by yougotwill — APPROVED** [review]
LGTM

**Pull request merged**